db/migrate/20181025230931_add_comments_to_vms_table.rb

```diff
@@ -0,0 +1,81 @@
+    change_column_comment :vms, :deprecated, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :description, "A custom description that users can add within the app."
+    change_column_comment :vms, :ems_created_on, "Date the associated EMS was created."
+    change_column_comment :vms, :ems_id, "ID of the ExtManagementSystem associated with the VM."
+    change_column_comment :vms, :evm_owner_id, "ID of VM owner from product database."
+    change_column_comment :vms, :ems_ref_obj, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :ems_cluster_id, "ID of associated cluster."
+    change_column_comment :vms, :fault_tolerance, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :flavor_id, "ID of associated flavor."
+    change_column_comment :vms, :format, "The format of the VM's disk, such as VMDK."
+    change_column_comment :vms, :guid, "A value used to uniquely identify the VM internally."
+    change_column_comment :vms, :id, "The internal database ID."
+    change_column_comment :vms, :host_id, "ID of the host that the VM/Template resides on."
+    change_column_comment :vms, :last_perf_capture_on, "Timestamp for last performance metrics capture."
+    change_column_comment :vms, :last_scan_on, "Timestamp for last SmartState Analysis."
+    change_column_comment :vms, :last_scan_attempt_on, "Timestamp for last attempt to perform SmartState Analysis."
+    change_column_comment :vms, :last_sync_on, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :linked_clone, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :location, "The location or region of the VM."
+    change_column_comment :vms, :memory_hot_add_enabled, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_hot_add_increment, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_hot_add_limit, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_limit, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_reserve, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_reserve_expand, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_shares, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :memory_shares_level, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :miq_group_id, "MIQ Group associated with the VM."
+    change_column_comment :vms, :name, "The name of the VM/Template."
+    change_column_comment :vms, :orchestration_stack_id, "ID of the orchestration stack associated with the VM."
+    change_column_comment :vms, :power_state, "The current power state, typically 'on' or 'off'."
+    change_column_comment :vms, :previous_state, "The previous power state change."
+    change_column_comment :vms, :publicly_available, "Indicates whether the VM is public or private."
+    change_column_comment :vms, :raw_power_state, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :registered, "Indicates whether the VM registered to a Provider."
```

Comment:
There's a column for this?  We have a method [`VmOrTemplate#registered?`](https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_or_template.rb#L311-L315) which checks template, ems_id, host_id

Reply:
And I don't see any of the parsers setting this.

Reply:
Should I mark it deprecated then?

Reply:
Yes